sources/core/Stride.Core.Design/CodeEditor/IDEType.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Stride.Core.CodeEditor;
+
+public enum IDEType
+{
+    VisualStudio, Rider, VSCode
```

Comment:
Nit: one line per value and documented (even if obvious).

Reply:
Done!